tests/test_operations.py

```diff
@@ -65,6 +61,30 @@ def test_delete(self, requests_mock):
         r = operations.delete(data_url)
         assert r is None
 
+    # test 'format' parameter
+    def test_create_format(self, data, requests_mock):
+        """Test create function parameter `format` overwriting `content-type`"""
+        requests_mock.post(base_url)
+        r = operations.create(base_url, data=data,
+            **{'headers': {'content-type': 'application/ld+json'}})
+        assert r is None
+
+    def test_read_format(self, data, requests_mock):
+        """Test read function parameter `format` overwriting `accept`"""
```

Comment:
I'm not seeing function parameter `format` used here anywhere?

Reply:
It's implicitly used with the default value. Will make it explicit.

Reply:
Yes, this is great. Now it's obvious that it tests whether the format argument correctly overwrites the header.